src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -470,7 +471,7 @@ private function addSubresourceOperation(bool $v3, array $subresourceOperation,
             $this->addPaginationParameters($v3, $resourceMetadata, $operationName, $pathOperation);
         }
 
-        return new \ArrayObject(['get' => $pathOperation]);
+        return $pathOperation;
```

Comment:
Shouldn't `ArrayObject` be kept?

```suggestion
        return new \ArrayObject($pathOperation);
```

Reply:
`$pathOperation` is already an `\ArrayObject`: https://github.com/api-platform/core/pull/3685/files#diff-32d63273c86a7e4c14c3fa6bb4131194R422

What is more, in my opinion, there was some inconsistency in `$paths`, because every subresource endpoint was added as `\ArrayObject` while other endpoints were a map `method => \ArrayObject`

Example from Sylius, before this change:
![image](https://user-images.githubusercontent.com/6140884/91418944-1b874180-e853-11ea-85cd-bab86c6bdc95.png)